blivet/formats/fs.py

```diff
@@ -1604,14 +1615,14 @@ def _setOptions(self, options):
 
     @property
     def free(self):
-        if self._mountpoint:
-            # If self._mountpoint is defined, it means this tmpfs mount
+        if self.systeMountpoint:
```

Comment:
Watch out for spelling mistakes!..."systeMountpoint". pylint will definitely catch these kinds of things.